homeassistant/components/luxor/config_flow.py

```diff
@@ -0,0 +1,126 @@
+"""Config flow for FX Luminaire Luxor low voltage controller integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import client_exceptions as aio_exceptions
+import luxor
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import aiohttp_client
+
+from .const import CONF_INCLUDE_LUXOR_THEMES, DEFAULT_INCLUDE_LUXOR_THEMES, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({CONF_HOST: str})
+
+
+async def validate_luxor(host: str, hass: HomeAssistant):
+    """Validate the host is a Luxor controller.
+
+    Also serves as a mock point for unit testing of forms.
+    """
```

Comment:
Testing should not affect code (or comments 😉 )
```suggestion
    """Validate the host is a Luxor controller."""
```